components/SectionPage.vue

```diff
@@ -18,8 +18,10 @@
             class="gothamist u-space--double--bottom mod-vertical mod-large"
             :show-gallery-icon="hasGallery(featuredStory.leadAsset)"
             :image="getArticleImage(featuredStory.leadAsset, featuredStory.ancestry[0].slug, featuredStory.listingImage)"
-            :image-height="150"
-            :image-width="150"
+            :image-height="560"
+            :image-width="413"
+            :image-max-height="560"
+            :image-max-width="413"
```

Comment:
image-max-height and image-max-width here are used when generating high dpi image sizes for the srcset. they should be the max dimensions of the original image, which should be available on the image object from the cms as `height` and `width`.

Reply:
and `image-width` and `image-height` are used for the base image size we want to display for the 1x image, and used to set width and height attributes on the image. I'm not sure we need to change them here unless the layout has changed